### ISSUE TITLE
Statistics for softmax and neural network with a softmax at the end

### ISSUE THREAD

**Issue by aromanro** [aromanro/MachineLearning#10]
Currently when computing stats, >0.5 is checked.

The code should be changed to pick up the output with the biggest value as being the prediction, instead of this test.
